docs/README.md

```diff
@@ -1,5 +1,46 @@
 # Kernl static documentation site
 
+## Targeting the Material for MkDocs Insiders version
```

Comment:
I'm not fond of telling in a public repository the name of our token, how we get our private version of m4mk and a way to create automaticly a new image in our private registry. Shall we make this part minimal (just the part with the version number and the warnings before upgrading), and put the full way of using m4mk in our blog repo ?

Reply:
@jonathlela You are absolutely right. I will correct.

What do you mean by blog repo?

Reply:
Done. Ready for review.